tests/unit/specs/store/connection.spec.js

```diff
@@ -86,24 +86,23 @@ describe(`Module: Connection`, () => {
     localStorage.clear()
   })
 
-  it(`assigns the user the fallback network if there is no persisted network 
+  it.only(`assigns the user the fallback network if there is no persisted network 
```

Comment:
attention

Reply:
```suggestion
  it(`assigns the user the fallback network if there is no persisted network 
```

Reply:
oops! Thank you. I always forget those 🤦‍♀